build-system/tasks/runtime-test/helpers-test-reporting.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const fetch = require('node-fetch');
+const fs = require('fs');
+const log = require('fancy-log');
+const JSON_REPORT_TEST_TYPES = new Set(['unit', 'integration']);
+const REPORTING_API_URL = 'https://amp-test-cases.uc.r.appspot.com/report';
+const {
+  travisBuildNumber,
+  travisJobNumber,
+  travisCommitSha,
+} = require('../../common/travis');
+
+function resultFilename(testType) {
+  return `results_${testType}.json`;
+}
+
+async function getReports() {
+  const reportPromises = JSON_REPORT_TEST_TYPES.map(async (testType) => {
+    const jsonString = await fs.readFile(resultFilename(testType));
+    const reportJson = JSON.parse(jsonString);
+    return {testType, reportJson};
+  });
+
+  const incompleteReports = await Promise.all(reportPromises);
+  const reports = incompleteReports.map(addJobAndBuildInfo);
+
+  return reports;
+}
+
+function addJobAndBuildInfo({testType, reportJson}) {
+  const build = {
```

Comment:
TODO: Change build type on amp-github-apps/test-case-reporting. We need to add the build number field, remove the pr number field, and reconsider the startedAt field.

Reply:
Done (see https://github.com/ampproject/amp-github-apps/pull/913).